lib/helper.js

```diff
@@ -209,6 +209,10 @@ class Helper {
     apiCoverage = new Map();
   }
 
+  static getNonSerializableMessage() {
+    return 'Non serializable object returned from evaluate';
```

Comment:
instead of returning a string, this should be a [Symbol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) exposed on the `puppeteer` namespace. This way you'll be able to do the following:

```js
const result = await page.evaluate(() => window);
if (result === puppeteer.NON_SERIALIZABLE_OBJECT) {
  // ...
}
```